app/Providers/ValidatorServiceProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+
+class ValidatorServiceProvider extends ServiceProvider {
+
+    public function boot()
+    {
+        $this->app['validator']->extend('Older60YearOlds', function ($attribute, $value, $parameters)
```

Comment:
You also have this logic in class: `Older60YearOlds`. Please pick 1 method to do custom validation.

Reply:
I got a problem that can not call rule file, so i decide to make a boot method, btw thanks